src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -328,6 +328,14 @@ TxFormats::TxFormats()
             {sfTicketSequence, soeOPTIONAL},
         },
         commonFields);
+
+    add(jss::Clawback,
+        ttCLAWBACK,
+        {
+            {sfAmount, soeREQUIRED},
+            {sfTicketSequence, soeOPTIONAL},
```

Comment:
I'm trying to think of use cases where someone would want to use the tickets feature with `Clawback`, but I'm drawing up a blank.

Since the owner directory needs to be empty for a valid `Clawback` transaction, I suspect there are not a whole lot of transactions which can be done before a `Clawback` right?